Makefile

```diff
@@ -479,8 +480,11 @@ bundle-push:
 api-docs: crdoc kustomize
 	@{ \
 	set -e ;\
+	TMP_MANIFEST_DIR=$$(mktemp -d) ; \
+	cp -r config/crd/* $$TMP_MANIFEST_DIR; \
+	$(MAKE) CRD_OPTIONS=$(CRD_OPTIONS),maxDescLen=800 MANIFEST_DIR=$$TMP_MANIFEST_DIR/bases manifests ;\
```

Comment:
Why here is `,maxDescLen=800` and the length is also defined in `CRD_OPTIONS ?= "crd:generateEmbeddedObjectMeta=true,maxDescLen=200"` ?

Reply:
This will call the `manifests` target using `CRD_OPTIONS="crd:generateEmbeddedObjectMeta=true,maxDescLen=200",maxDescLen=800`. The second `maxDescLen` overrides the first one. By doing this, we can:
- Reuse the value of `CRD_OPTIONS`
- Use the new value for `maxDescLen` without polluting the bundle

Reply:
Do we actually need the max length for the api-docs?